src/front/ticketer/lib/backend_communication/model/event_status.dart

```diff
@@ -0,0 +1,11 @@
+enum EventStatus {
+  Unverified,
+  Verified,
+  Published,
+  Opened,
+  Closed,
+  Finished,
+  Cancelled,
+  Held,
+  Recalled,
+}
```

Comment:
Add new line

Reply:
Also add Strings to nicely display statuses. This will be useful later. Do this the same as ResponseCode enum

Reply:
Noted